bin/fun-local-start.js

```diff
@@ -37,6 +37,13 @@ if (program.args.length > 1) {
   program.help();
 }
 
+if (program.debugPort && program.args.length === 0) {
+
+  console.error();
+  console.error('  error: missing argument <[service/]function>\n');
```

Comment:
Must specify the '<[service/]function>' parameter when using '-d, --debug-port <port>' option.
For example, you can use 'fun local start -d 3000 functionName' to start your function locally with debugging mode.

Reply:
done